llvm/docs/LangRef.rst

```diff
@@ -16464,21 +16464,13 @@ type.
 
 Semantics:
 """"""""""
+Follows the IEEE754 2008 semantics for minNum, except for handling of
++0.0 vs -0.0. This matches the behavior of libm's fmin.
```

Comment:
IEEE 754 has the same indifference to the result of `minNum(-0.0, +0.0)`, it's not a deviation from the IEEE 754 specification.